app/src/main/java/org/kiwix/kiwixmobile/localFileTransfer/WifiDirectManager.kt

```diff
@@ -83,23 +83,19 @@ class WifiDirectManager @Inject constructor(
   // For receiving the broadcasts given by above filter
   private lateinit var receiver: BroadcastReceiver
 
-  // Corresponds to P2P group formed between the two devices
-  private lateinit var groupInfo: WifiP2pInfo
   private lateinit var senderSelectedPeerDevice: WifiP2pDevice
```

Comment:
this is also a pointless variable, the original variable does not need to be stored, just pass it to `connect`